src/Resources/config/services.yaml

```diff
@@ -1,4 +1,8 @@
 parameters:
+  monsieurbiz.search.elasticsearch.host: '%env(default::MONSIEURBIZ_SEARCHPLUGIN_ES_HOST)%'
+  monsieurbiz.search.elasticsearch.port: '%env(default::MONSIEURBIZ_SEARCHPLUGIN_ES_PORT)%'
+  monsieurbiz.search.elasticsearch.fallback_url: 'http://%monsieurbiz.search.elasticsearch.host%:%monsieurbiz.search.elasticsearch.port%/'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏http or https ? Do we need to be able to manage it ?